modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/resource/v1_0/KeywordResourceImpl.java

```diff
@@ -187,6 +187,23 @@ public Keyword putKeyword(Long keywordId, Keyword keyword)
 			_assetTagService.updateTag(keywordId, keyword.getName(), null));
 	}
 
+	@Override
+	protected Long getPermissionCheckerGroupId(Object id) throws Exception {
+		AssetTag assetTag = _assetTagService.getTag((Long)id);
+
+		return assetTag.getGroupId();
+	}
+
+	@Override
+	protected String getPermissionCheckerPortletName(Object id) {
+		return "com.liferay.asset.tags";
+	}
+
+	@Override
+	protected String getPermissionCheckerResourceName(Object id) {
+		return "com.liferay.asset.tags";
```

Comment:
is it the same in both?

Reply:
It was weird to me also but making some empirical tests and checking database it seemed to me that the individual entities can't manage permissions.

I will double check, because now I'm not completely sure about this

Reply:
Ok!

Reply:
I've confirmed there is no permission management at AssetTag level:
https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/resource-actions/asset.xml